governance.md

```diff
@@ -59,260 +70,170 @@ The foundations of Project governance are:
-  Team.
-- This Subcommittee shall NOT make decisions about the direction, scope or
-  technical direction of the Project.
-- This Subcommittee will have at least 5 members. No more than 2 Subcommittee
-  Members can report to one person (either directly or indirectly) through
-  employment or contracting work (including the reportee, i.e. the reportee + 1
-  is the max). This avoids effective majorities resting on one person.
-
 ## Institutional Partners and Funding
 
-The BDFL and Core Team are the primary leadership for the project. No outside
+The Stewards are the primary leaders of the project. No outside
 institution, individual or legal entity has the ability to own, control, usurp
 or influence the project other than by participating in the Project as
-Contributors and Core Team. However, because institutions are the primary
+Contributors and Stewards. However, because institutions can be an important
 funding mechanism for the project, it is important to formally acknowledge
 institutional participation in the project. These are Institutional Partners.
-
 An Institutional Contributor is any individual Project Contributor who
 contributes to the project as part of their official duties at an Institutional
-Partner. Likewise, an Institutional Core Team Member is any Core Team Member
+Partner. Likewise, an Institutional Steward is any Steward
 who contributes to the project as part of their official duties at an
 Institutional Partner.
-
 With these definitions, an Institutional Partner is any recognized legal entity
 in the United States or elsewhere that employs at least one Institutional
-Contributor or Institutional Core Team Member. Institutional Partners can be
+Contributor or Institutional Steward. Institutional Partners can be
 for-profit or non-profit entities.
-
 Institutions become eligible to become an Institutional Partner by employing
 individuals who actively contribute to The Project as part of their official
 duties. To state this another way, the only way for an Institutional Partner to
 influence the project is by actively contributing to the open development of
 the project, on equal terms with any other member of the community of
-Contributors and Core Team Members. Merely using pandas Software or Services in
+Contributors and Stewards. Merely using Project Software in
 an institutional context does not allow an entity to become an Institutional
-Partner. Financial gifts do not enable an entity to become an Institutional
+Partner. Financial gifts, which are recognized on the _pandas_ web site,
+do not enable an entity to become an Institutional
 Partner. Once an institution becomes eligible for Institutional Partnership,
-the Core Team must nominate and approve the Partnership.
-
-If an existing Institutional Partner no longer has a contributing employee,
-they will be given a one-year grace period for other employees to begin
-contributing.
+the Steering Committee must nominate and approve the Partnership.
```

Comment:
Seems odd that a smaller steering committee has the ability to approve institutional partners  without say/informing those working on the project. Maybe stewards/ Core Lib Team members should have the ability to object?

Reply:
We will clarify that the Steering Committee will solicit input from the Stewards before making decisions, which would cover this case.

Reply:
Also, such changes will normally always happen in the open through PRs (typically on this repo), and so every member of the project will be able to see those changes and comment on it.

Reply:
fixed in next revision